script/Ciel/Day 13/13_01_01_CIEL13_4.txt

```diff
@@ -388,22 +419,24 @@ I've always wanted to be a student like the two of you.'
 [sha:0365d4fa364ac94327121fc30b1bcd0f90a0d9cc]{
 -- Page 334, Offset 34171.
 -- でも、あの人自体が嘘だった。
-Even then, her very existence is built on lies.
+Even so, her very existence is built on lies.
 }
 [sha:c2347e0e639baaeda27b34d5c29eaf28a6038610]{
 -- Page 334, Offset 34172.
 -- あの笑顔が本当の嘘とは俺には思えない。
-I can't bring myself to believe that her smile was nothing more than a lie.
+// lint-off:LintDupedWord
+I can't bring myself to believe that that smile of hers was fake.
```

Comment:
Could use a conjunction.
Also, I think the repetition of lie here is fine, and appropriate, if you add a bit 

>Regardless, I find it hard to believe her smile was part of it [the lie].

Reply:
Re: conjunction, I feel like this line ties more with the following than the preceding, plus it feels a bit awkward for the first _and_ second lines on that page to start with a conjunction like that - 
> Even so...
> Regardless...